poutyne/framework/model.py

```diff
@@ -27,11 +27,16 @@ class Model:
 
     Args:
         model (torch.nn.Module): A PyTorch module.
-        optimizer (torch.optim.Optimizer): Initialized PyTorch optimizer.
-        loss_function: Loss function. It can be any PyTorch loss layer or custom loss function. It
-            can also be a string with the same name as a PyTorch loss function (either the functional or
-            object name). The loss function must have the signature ``loss_function(input, target)``
-            where ``input`` is the prediction of the network and ``target`` is the ground truth.
+        optimizer (Union[torch.optim.Optimizer, str]): If torch.optim.Optimier, an initialized PyTorch.
+            If str, should be the optimizer's name in Pytorch (i.e. 'Adam' for torch.optim.Adam).
+            (Default value = 'sgd')
+        loss_function(Union[Callable, str]) It can be any PyTorch
+            loss layer or custom loss function. It can also be a string with the same name as a PyTorch
+            loss function (either the functional or object name). The loss function must have the signature
+            ``loss_function(input, target)`` where ``input`` is the prediction of the network and ``target``
+            is the ground truth. If ``None``, will default to, in priority order, either the model's own
+            loss function or the default loss function associated with the ``exp_type``.
```

Comment:
``exp_type```(i.e. what is now named ``task`` should not appear here.

Reply:
Actually, I think this last sentence is not true for Model.

Reply:
```suggestion
            loss function or the default loss function associated with the ``task``.
```

Reply:
Last sentence is the true behaviour of Experiment._get_loss_function(). If ``loss_function`` is None, we first look if the module has one, else we assign it with the task's default value.

Reply:
But this is in Model not Experiment.

Reply:
I am retarded. Fixed by 8c3064c6769cb9b03b558cb571e6ca696c8f32bd.